babel.config.js

```diff
@@ -5,7 +5,11 @@ module.exports = {
   ],
   plugins: [
     [require.resolve('@babel/plugin-transform-runtime'), { corejs: 3 }],
-    [require.resolve('babel-plugin-transform-inline-environment-variables')],
+
+    require.resolve('babel-plugin-transform-inline-environment-variables'),
+
+    require.resolve('babel-plugin-lodash'),
```

Comment:
Is this still required?

Reply:
I included it because why not, even the Serverless deployment doesn't need to be bigger than it has to :sweat: